ember-mirage/src/test-support/setup-mirage.js

```diff
@@ -28,6 +28,11 @@ export function setupMirage(hooks = self, { createServer, config }) {
     }
 
     this.server = await createServer(config ?? {});
+
+    // This only works after the server was created.
+    this.server.logging =
```

Comment:
I've been enabling logging via the call site.

For someone who is setting logging themselves, how do we not stomp on them?

Maybe, only set if the qp is set?

Reply:
Yeah this is something I was wondering about as well (and in part why this was a draft until just now 😂 ). It did seem that, when I was setting this up anyway, logging was enabled _by default_ when not explicitly passing false, so only setting it when the QP is set wouldn't work as far as I could see.

Reply:
While we figure this out, let me split the ember-qunit peerdep into a separate PR as that's required regardless to fix the existing bit of code.